mesa/visualization/ModularVisualization.py

```diff
@@ -341,6 +341,9 @@ def reset_model(self):
                 model_params[key] = val
 
         self.model = self.model_cls(**model_params)
+        # We specify the `running` attribute her so that the user doesn't have
```

Comment:
should be here not her...but otherwise looks good

Reply:
Ugh, fixed!